packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/components/checkDate.ts

```diff
@@ -0,0 +1,17 @@
+
+export const checkDate = () => {
```

Comment:
This function should take the date as a param and then each function should reference that instead of having to pass a date into every function...

At least that what I'd expect. Or if not then each of these methods should be exported separately, no need for this constructor

Reply:
[db3f9c7](https://github.com/astrolicious/studiocms/pull/125/commits/db3f9c76777ac7d5c96023d27ca4f378dfa98af7) like this?